Client/mods/deathmatch/logic/CClientVectorGraphic.h

```diff
@@ -0,0 +1,53 @@
+/*****************************************************************************
+ *
+ *  PROJECT:     Multi Theft Auto
+ *  LICENSE:     See LICENSE in the top level directory
+ *
+ *  Multi Theft Auto is available from http://www.multitheftauto.com/
+ *
+ *****************************************************************************/
+
+#pragma once
+
+#include <svgdocument.h>
```

Comment:
Since you only store a pointer to the SVG document, you could avoid pulling in the library header in this header file.

Reply:
I think I need this header, it complained when I took it out due to the `GetSVGDocumentXML`

Reply:
You could move the inline definition to the soruce file to avoid the header inclusion.